# Frequency of the status replies in Hz
float32 freq_front_left
float32 freq_front_right
float32 freq_rear_left
float32 freq_rear_right